[2024-03-20T17:31:42.789+0000] {processor.py:161} INFO - Started process (PID=76) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:31:42.793+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:31:42.807+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:31:42.798+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:31:42.856+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:31:42.850+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 11, in <module>
    print_welcome_task = PythonOperator(
NameError: name 'PythonOperator' is not defined
[2024-03-20T17:31:42.876+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:31:42.976+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.206 seconds
[2024-03-20T17:32:13.849+0000] {processor.py:161} INFO - Started process (PID=77) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:32:13.850+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:32:13.852+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:32:13.852+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:32:13.863+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:32:13.861+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 11, in <module>
    print_welcome_task = PythonOperator(
NameError: name 'PythonOperator' is not defined
[2024-03-20T17:32:13.864+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:32:13.907+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.062 seconds
[2024-03-20T17:32:44.239+0000] {processor.py:161} INFO - Started process (PID=78) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:32:44.241+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:32:44.243+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:32:44.243+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:32:44.252+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:32:44.250+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 11, in <module>
    print_welcome_task = PythonOperator(
NameError: name 'PythonOperator' is not defined
[2024-03-20T17:32:44.253+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:32:44.299+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.063 seconds
[2024-03-20T17:32:57.856+0000] {processor.py:161} INFO - Started process (PID=79) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:32:57.859+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:32:57.861+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:32:57.861+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:32:57.876+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:32:57.874+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 14, in <module>
    dag = DAG(
NameError: name 'DAG' is not defined
[2024-03-20T17:32:57.877+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:32:57.922+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.070 seconds
[2024-03-20T17:33:28.740+0000] {processor.py:161} INFO - Started process (PID=80) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:33:28.743+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:33:28.745+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:33:28.745+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:33:28.756+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:33:28.754+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 14, in <module>
    dag = DAG(
NameError: name 'DAG' is not defined
[2024-03-20T17:33:28.758+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:33:28.805+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.069 seconds
[2024-03-20T17:33:59.461+0000] {processor.py:161} INFO - Started process (PID=81) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:33:59.463+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:33:59.466+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:33:59.465+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:33:59.475+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:33:59.473+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 14, in <module>
    dag = DAG(
NameError: name 'DAG' is not defined
[2024-03-20T17:33:59.477+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:33:59.525+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.068 seconds
[2024-03-20T17:34:30.231+0000] {processor.py:161} INFO - Started process (PID=82) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:34:30.234+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:34:30.236+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:34:30.235+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:34:30.247+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:34:30.245+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 14, in <module>
    dag = DAG(
NameError: name 'DAG' is not defined
[2024-03-20T17:34:30.249+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:34:30.297+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.070 seconds
[2024-03-20T17:34:50.629+0000] {processor.py:161} INFO - Started process (PID=83) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:34:50.632+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:34:50.633+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:34:50.633+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:34:50.799+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:34:51.279+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:34:51.278+0000] {override.py:1769} INFO - Created Permission View: can edit on DAG:welcome_dag
[2024-03-20T17:34:51.349+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:34:51.349+0000] {override.py:1769} INFO - Created Permission View: can delete on DAG:welcome_dag
[2024-03-20T17:34:51.405+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:34:51.404+0000] {override.py:1769} INFO - Created Permission View: can read on DAG:welcome_dag
[2024-03-20T17:34:51.406+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:34:51.406+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:34:51.439+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:34:51.438+0000] {dag.py:3069} INFO - Creating ORM DAG for welcome_dag
[2024-03-20T17:34:51.472+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:34:51.471+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:34:51.599+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.974 seconds
[2024-03-20T17:35:15.057+0000] {processor.py:161} INFO - Started process (PID=84) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:35:15.060+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:35:15.061+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:35:15.061+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:35:15.094+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:35:15.110+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:35:15.110+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:35:15.147+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:35:15.146+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:35:15.234+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.181 seconds
[2024-03-20T17:35:26.648+0000] {processor.py:161} INFO - Started process (PID=85) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:35:26.651+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:35:26.653+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:35:26.652+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:35:26.683+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:35:26.718+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:35:26.718+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:35:26.753+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:35:26.753+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:35:26.842+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.198 seconds
[2024-03-20T17:35:57.459+0000] {processor.py:161} INFO - Started process (PID=86) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:35:57.461+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:35:57.464+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:35:57.463+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:35:57.488+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:35:57.525+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:35:57.524+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:35:57.563+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:35:57.563+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:35:57.634+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.179 seconds
[2024-03-20T17:36:28.779+0000] {processor.py:161} INFO - Started process (PID=107) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:36:28.781+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:36:28.783+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:36:28.782+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:36:28.804+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:36:28.837+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:36:28.837+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:36:28.872+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:36:28.871+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:36:28.938+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.163 seconds
[2024-03-20T17:36:52.589+0000] {processor.py:161} INFO - Started process (PID=108) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:36:52.592+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:36:52.594+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:36:52.594+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:36:52.619+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:36:52.616+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:36:52.621+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:36:52.685+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.099 seconds
[2024-03-20T17:36:54.526+0000] {processor.py:161} INFO - Started process (PID=109) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:36:54.529+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:36:54.530+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:36:54.530+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:36:54.562+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:36:54.559+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:36:54.565+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:36:54.609+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.087 seconds
[2024-03-20T17:37:25.646+0000] {processor.py:161} INFO - Started process (PID=110) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:37:25.648+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:37:25.650+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:37:25.650+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:37:25.675+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:37:25.671+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:37:25.677+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:37:25.735+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.094 seconds
[2024-03-20T17:37:46.177+0000] {processor.py:161} INFO - Started process (PID=111) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:37:46.180+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:37:46.183+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:37:46.182+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:37:46.211+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:37:46.209+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:37:46.213+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:37:46.262+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.090 seconds
[2024-03-20T17:38:16.904+0000] {processor.py:161} INFO - Started process (PID=112) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:38:16.907+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:38:16.908+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:38:16.908+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:38:16.934+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:38:16.931+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:38:16.936+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:38:16.977+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.077 seconds
[2024-03-20T17:38:47.755+0000] {processor.py:161} INFO - Started process (PID=113) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:38:47.757+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:38:47.759+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:38:47.759+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:38:47.780+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:38:47.778+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:38:47.782+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:38:47.824+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.073 seconds
[2024-03-20T17:39:18.516+0000] {processor.py:161} INFO - Started process (PID=114) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:39:18.519+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:39:18.522+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:39:18.521+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:39:18.541+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:39:18.539+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:39:18.542+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:39:18.587+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.075 seconds
[2024-03-20T17:39:49.049+0000] {processor.py:161} INFO - Started process (PID=115) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:39:49.065+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:39:49.094+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:39:49.079+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:39:49.430+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:39:49.416+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:39:49.451+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:39:49.610+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.620 seconds
[2024-03-20T17:40:19.839+0000] {processor.py:161} INFO - Started process (PID=116) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:40:19.842+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:40:19.844+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:40:19.843+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:40:19.865+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:40:19.863+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:40:19.866+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:40:19.909+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.074 seconds
[2024-03-20T17:40:50.267+0000] {processor.py:161} INFO - Started process (PID=117) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:40:50.269+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:40:50.271+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:40:50.271+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:40:50.293+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:40:50.291+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:40:50.295+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:40:50.336+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.073 seconds
[2024-03-20T17:41:20.624+0000] {processor.py:161} INFO - Started process (PID=118) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:41:20.627+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:41:20.629+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:41:20.629+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:41:20.650+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:41:20.648+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:41:20.652+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:41:20.693+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.073 seconds
[2024-03-20T17:41:51.024+0000] {processor.py:161} INFO - Started process (PID=119) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:41:51.027+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:41:51.030+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:41:51.029+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:41:51.050+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:41:51.048+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:41:51.052+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:41:51.094+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.074 seconds
[2024-03-20T17:42:21.421+0000] {processor.py:161} INFO - Started process (PID=120) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:42:21.424+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:42:21.426+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:42:21.426+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:42:21.445+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:42:21.444+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:42:21.447+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:42:21.488+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.071 seconds
[2024-03-20T17:42:51.783+0000] {processor.py:161} INFO - Started process (PID=121) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:42:51.785+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:42:51.787+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:42:51.787+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:42:51.807+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:42:51.805+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:42:51.808+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:42:51.854+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.075 seconds
[2024-03-20T17:43:22.131+0000] {processor.py:161} INFO - Started process (PID=122) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:43:22.135+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:43:22.137+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:43:22.136+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:43:22.156+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:43:22.154+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:43:22.157+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:43:22.200+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.073 seconds
[2024-03-20T17:43:52.484+0000] {processor.py:161} INFO - Started process (PID=123) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:43:52.487+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:43:52.489+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:43:52.488+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:43:52.508+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:43:52.505+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:43:52.510+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:43:52.558+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.077 seconds
[2024-03-20T17:44:22.670+0000] {processor.py:161} INFO - Started process (PID=124) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:44:22.673+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:44:22.674+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:44:22.674+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:44:22.693+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:44:22.691+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:44:22.694+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:44:22.736+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.069 seconds
[2024-03-20T17:44:53.136+0000] {processor.py:161} INFO - Started process (PID=125) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:44:53.140+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:44:53.142+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:44:53.141+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:44:53.161+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:44:53.159+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:44:53.162+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:44:53.203+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.071 seconds
[2024-03-20T17:45:23.505+0000] {processor.py:161} INFO - Started process (PID=126) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:45:23.507+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:45:23.510+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:45:23.509+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:45:23.514+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:45:23.512+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:45:23.516+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:45:23.555+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.068 seconds
[2024-03-20T17:45:53.614+0000] {processor.py:161} INFO - Started process (PID=127) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:45:53.616+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:45:53.618+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:45:53.618+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:45:53.637+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:45:53.635+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:45:53.639+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:45:53.678+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.068 seconds
[2024-03-20T17:46:24.121+0000] {processor.py:161} INFO - Started process (PID=128) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:46:24.123+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:46:24.125+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:46:24.125+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:46:24.147+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:46:24.143+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:46:24.150+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:46:24.197+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.080 seconds
[2024-03-20T17:46:49.285+0000] {processor.py:161} INFO - Started process (PID=129) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:46:49.287+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:46:49.289+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:46:49.289+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:46:49.313+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:46:49.310+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 44, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:46:49.315+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:46:49.359+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.077 seconds
[2024-03-20T17:47:05.478+0000] {processor.py:161} INFO - Started process (PID=130) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:47:05.480+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:47:05.482+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:47:05.481+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:47:05.508+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:47:05.506+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 42, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:47:05.510+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:47:05.552+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.078 seconds
[2024-03-20T17:47:35.824+0000] {processor.py:161} INFO - Started process (PID=131) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:47:35.827+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:47:35.829+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:47:35.829+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:47:35.856+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:47:35.853+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 42, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:47:35.857+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:47:35.904+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.086 seconds
[2024-03-20T17:48:06.096+0000] {processor.py:161} INFO - Started process (PID=132) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:48:06.099+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:48:06.104+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:48:06.103+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:48:06.132+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:48:06.129+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 42, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:48:06.134+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:48:06.193+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.102 seconds
[2024-03-20T17:48:36.382+0000] {processor.py:161} INFO - Started process (PID=133) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:48:36.385+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:48:36.386+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:48:36.386+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:48:36.406+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:48:36.403+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 42, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:48:36.408+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:48:36.456+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.078 seconds
[2024-03-20T17:49:06.592+0000] {processor.py:161} INFO - Started process (PID=134) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:49:06.595+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:49:06.596+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:49:06.596+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:49:06.617+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:49:06.615+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 42, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:49:06.620+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:49:06.664+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.077 seconds
[2024-03-20T17:49:36.828+0000] {processor.py:161} INFO - Started process (PID=135) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:49:36.831+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:49:36.833+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:49:36.832+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:49:36.851+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:49:36.849+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/welcome-dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/welcome-dag.py", line 42, in <module>
    print_welcome_task >> print_date >> print_random_quote
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 104, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 262, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskmixin.py", line 214, in _set_relatives
    task_object.update_relative(self, not upstream, edge_modifier=edge_modifier)
AttributeError: 'function' object has no attribute 'update_relative'
[2024-03-20T17:49:36.853+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:49:36.895+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.071 seconds
[2024-03-20T17:49:39.655+0000] {processor.py:161} INFO - Started process (PID=136) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:49:39.658+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:49:39.660+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:49:39.660+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:49:39.723+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:49:40.083+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:49:40.082+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:49:40.131+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:49:40.131+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:49:40.244+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.594 seconds
[2024-03-20T17:49:50.630+0000] {processor.py:161} INFO - Started process (PID=137) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:49:50.633+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:49:50.635+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:49:50.634+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:49:50.667+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:49:50.682+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:49:50.682+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:49:50.718+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:49:50.718+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:49:50.792+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.167 seconds
[2024-03-20T17:50:21.572+0000] {processor.py:161} INFO - Started process (PID=138) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:50:21.575+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:50:21.577+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:50:21.577+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:50:21.600+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:50:21.630+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:50:21.630+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:50:21.664+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:50:21.664+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:50:21.725+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.157 seconds
[2024-03-20T17:50:23.999+0000] {processor.py:161} INFO - Started process (PID=139) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:50:24.001+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:50:24.003+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:50:24.003+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:50:24.033+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:50:24.071+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:50:24.071+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:50:24.106+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:50:24.106+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:50:24.177+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.181 seconds
[2024-03-20T17:50:39.459+0000] {processor.py:161} INFO - Started process (PID=140) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:50:39.462+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:50:39.465+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:50:39.465+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:50:39.499+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:50:39.535+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:50:39.535+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:50:39.572+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:50:39.572+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:50:39.648+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.196 seconds
[2024-03-20T17:51:10.302+0000] {processor.py:161} INFO - Started process (PID=141) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:51:10.305+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:51:10.307+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:51:10.307+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:51:10.333+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:51:10.372+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:51:10.371+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:51:10.413+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:51:10.412+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:51:10.499+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.202 seconds
[2024-03-20T17:51:41.093+0000] {processor.py:161} INFO - Started process (PID=142) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:51:41.097+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:51:41.099+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:51:41.098+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:51:41.122+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:51:41.157+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:51:41.156+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:51:41.192+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:51:41.191+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:51:41.263+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.174 seconds
[2024-03-20T17:52:11.844+0000] {processor.py:161} INFO - Started process (PID=143) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:52:11.847+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:52:11.849+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:52:11.849+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:52:11.870+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:52:11.903+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:52:11.903+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:52:11.937+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:52:11.937+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:52:12.008+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.168 seconds
[2024-03-20T17:52:19.842+0000] {processor.py:161} INFO - Started process (PID=144) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:52:19.860+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:52:19.862+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:52:19.861+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:52:19.888+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:52:19.927+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:52:19.927+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:52:19.963+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:52:19.963+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:52:20.045+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.207 seconds
[2024-03-20T17:52:50.516+0000] {processor.py:161} INFO - Started process (PID=145) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:52:50.518+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:52:50.520+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:52:50.520+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:52:50.544+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:52:50.579+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:52:50.578+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:52:50.611+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:52:50.611+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:52:50.685+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.174 seconds
[2024-03-20T17:53:00.656+0000] {processor.py:161} INFO - Started process (PID=146) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:53:00.659+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:53:00.662+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:53:00.661+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:53:00.696+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:53:00.735+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:53:00.735+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:53:00.776+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:53:00.776+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:53:00.855+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.204 seconds
[2024-03-20T17:53:31.957+0000] {processor.py:161} INFO - Started process (PID=147) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:53:31.961+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:53:31.964+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:53:31.963+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:53:31.991+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:53:32.035+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:53:32.035+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:53:32.071+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:53:32.071+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:53:32.137+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.184 seconds
[2024-03-20T17:54:02.895+0000] {processor.py:161} INFO - Started process (PID=148) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:54:02.898+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:54:02.899+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:54:02.899+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:54:02.921+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:54:02.955+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:54:02.955+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:54:02.987+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:54:02.987+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:54:03.054+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.163 seconds
[2024-03-20T17:54:33.293+0000] {processor.py:161} INFO - Started process (PID=149) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:54:33.297+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:54:33.299+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:54:33.298+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:54:33.320+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:54:33.356+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:54:33.356+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:54:33.394+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:54:33.394+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:54:33.466+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.177 seconds
[2024-03-20T17:55:03.815+0000] {processor.py:161} INFO - Started process (PID=150) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:55:03.818+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:55:03.820+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:55:03.819+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:55:03.841+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:55:03.875+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:55:03.874+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:55:03.911+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:55:03.910+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:55:04.020+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.209 seconds
[2024-03-20T17:55:34.787+0000] {processor.py:161} INFO - Started process (PID=151) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:55:34.790+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:55:34.792+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:55:34.792+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:55:34.814+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:55:34.851+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:55:34.851+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:55:34.887+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:55:34.887+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:55:34.958+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.176 seconds
[2024-03-20T17:56:05.379+0000] {processor.py:161} INFO - Started process (PID=182) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:56:05.382+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:56:05.384+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:56:05.383+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:56:05.419+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:56:05.472+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:56:05.472+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:56:05.509+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:56:05.509+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:56:05.582+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.208 seconds
[2024-03-20T17:56:36.744+0000] {processor.py:161} INFO - Started process (PID=183) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:56:36.747+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:56:36.749+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:56:36.748+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:56:36.772+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:56:36.808+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:56:36.808+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:56:36.845+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:56:36.845+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:56:36.937+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.197 seconds
[2024-03-20T17:57:08.081+0000] {processor.py:161} INFO - Started process (PID=184) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:57:08.083+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:57:08.085+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:57:08.085+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:57:08.107+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:57:08.140+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:57:08.140+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:57:08.172+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:57:08.171+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:57:08.233+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.157 seconds
[2024-03-20T17:57:38.368+0000] {processor.py:161} INFO - Started process (PID=185) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:57:38.371+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:57:38.374+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:57:38.373+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:57:38.397+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:57:38.477+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:57:38.476+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:57:38.549+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:57:38.549+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:57:38.650+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.286 seconds
[2024-03-20T17:58:08.956+0000] {processor.py:161} INFO - Started process (PID=186) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:58:08.959+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:58:08.961+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:58:08.960+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:58:08.984+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:58:09.018+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:58:09.017+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:58:09.056+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:58:09.056+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:58:09.119+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.167 seconds
[2024-03-20T17:58:39.600+0000] {processor.py:161} INFO - Started process (PID=187) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:58:39.607+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:58:39.616+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:58:39.613+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:58:39.691+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:58:39.780+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:58:39.780+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:58:39.862+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:58:39.861+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:58:40.047+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.461 seconds
[2024-03-20T17:59:10.594+0000] {processor.py:161} INFO - Started process (PID=188) to work on /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:59:10.597+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/welcome-dag.py for tasks to queue
[2024-03-20T17:59:10.599+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:59:10.598+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:59:10.621+0000] {processor.py:840} INFO - DAG(s) 'welcome_dag' retrieved from /opt/airflow/dags/welcome-dag.py
[2024-03-20T17:59:10.664+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:59:10.664+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-03-20T17:59:10.700+0000] {logging_mixin.py:188} INFO - [2024-03-20T17:59:10.700+0000] {dag.py:3834} INFO - Setting next_dagrun for welcome_dag to 2024-03-19 23:00:00+00:00, run_after=2024-03-20 23:00:00+00:00
[2024-03-20T17:59:10.774+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/welcome-dag.py took 0.185 seconds
